workers/email/src/templates/emails/events-limit-almost-reached/html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends '../../components/layout.twig' %}
+
+{% block header %}
+  {% include '../../components/workspace.twig' with {workspace: workspace} %}
+{% endblock %}
+
+{% block content %}
+  <tr>
+    <td>
+      <img src="{{ hostOfStatic }}/email/low-balance-icon.png" width="32" height="32" hspace="3" style="display: block; vertical-align: middle; margin: 23px auto 0;">
+    </td>
+  </tr>
+  <tr>
+    <td align="center" style="padding: 15px 0;">
+      <font color="#dbe6ff" style="font-size: 15px; text-align: center; color: #dbe6ff; letter-spacing: 0.4px;">
+        <span style="vertical-align: middle; display: inline-block;">
+          Your plan limit is almost reached
+        </span>
+      </font>
+    </td>
+  </tr>
+  <tr>
+    <td style="display: block; padding: 20px; margin-bottom: 30px; border-width: 1px; border-color: #494f5e; border-style: solid; border-radius: 10px; line-height: 1.47">
+      <font color="#dbe6ff" style="font-size: 15px; letter-spacing: 0.4px;">
+        {{ eventsCount }} out of {{ eventsLimit }} events used to the plan ending for workspace {{ workspace.name | escape }}!
```

Comment:
You've used 9532 of 10000 events available on CodeX workspace plan. Upgrade the plan and don't miss the new ones.